.github/workflows/publish.yaml

```diff
@@ -24,22 +24,39 @@ jobs:
         with:
           strip_v: true
 
+  get-previous-tag:
+    runs-on: ubuntu-22.04
+    outputs:
+      previous-tag: ${{ steps.previous-tag.outputs.tag }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Get previous tag
+        id: previous-tag
+        run: |
+          git fetch -a
+          tags=($(git tag --sort="-committerdate"))
+          previous_tag=${tags[1]}
+          echo "::set-output name=tag::$previous_tag"
+
   generate-release-notes-pr:
     runs-on: ubuntu-22.04
-    needs: [get-tag]
+    needs: [get-tag, get-previous-tag]
     if: github.ref_type != 'branch'
     steps:
     - name: Checkout
       uses: actions/checkout@v3
 
     - name: Generate Release Notes PR
       env:
+        GIT_PREV_TAG: ${{ needs.get-previous-tag.outputs.previous-tag }}
```

Comment:
the output name above is `tag`.  we can change one or the other so they match

Reply:
I updated both to use `tag` instead